pkg/validator/schema.go

```diff
@@ -251,6 +253,18 @@ func applySchemaCheck(conf *config.Configuration, checkID string, test schemaTes
 	} else if check.Target == config.TargetPod {
 		passes, issues, err = check.CheckPod(test.Resource.PodSpec)
 	} else if check.Target == config.TargetContainer {
+		containerIndex := funk.IndexOf(test.Resource.PodSpec.Containers, func(value corev1.Container) bool {
+			return value.Name == test.Container.Name
+		})
+		if test.Resource.Kind == "CronJob" {
+			prefix = "/spec/jobTemplate/spec/template/spec/containers/" + strconv.Itoa(containerIndex)
+		} else if test.Resource.Kind == "Pod" {
+			// Pod
+			prefix = "/spec/containers/" + strconv.Itoa(containerIndex)
+		} else {
+			// Job, Deployment, Daemonset, ReplicationController and Statefulset
+			prefix = "/spec/template/spec/containers/" + strconv.Itoa(containerIndex)
+		}
```

Comment:
we need similar logic above for `check.Target == TargetPod`

Looks like the if/else logic is kind of complicated. We might want to start a new conditional that says
```go
if check.Target == TargetContainer {
  // logic you have above
} else if check.Target == TargetPod {
  // similar logic, but removing /containers/* from the end
}
```